geode-core/src/main/java/org/apache/geode/distributed/internal/locks/DLockRequestProcessor.java

```diff
@@ -196,6 +196,13 @@ long getLeaseExpireTime() {
     return this.response.leaseExpireTime;
   }
 
+  /**
+   *
+   * @param interruptible
+   * @param lockId
+   * @return
+   * @throws InterruptedException only possible if interruptible is true.
+   */
   protected boolean requestLock(boolean interruptible, int lockId) throws InterruptedException {
     final boolean isDebugEnabled_DLS = logger.isTraceEnabled(LogMarker.DLS);
```

Comment:
The `LogMarker` is used in the `logger.trace` function. This is based on an old pattern that was written because an old version of the logger was slow, so it was better to gate on a boolean for performance. The logging function will do an identical check to this one before printing a log message.